src/payment/strategies/apple-pay/apple-pay-payment-initialize-options.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * A set of options that are required to initialize the Applepay payment method with:
+ *
+ * 1) ApplePay:
+ *
+ * ```js
+ * service.initializePayment({
+ *     methodId: 'applepay',
+ *     applepay: {
+ *         shippingLabel: 'Shipping',
+ *         subtotalLabel: 'Sub total',
+ *     }
+ * });
+ * ```
+ */
+ export default interface ApplePayPaymentInitializeOptions {
+    /**
+     * Shipping label to be passed to apple sheet.
+     */
+     shippingLabel: string;
```

Comment:
I think this could be optional as we can provide a default value if not provided.